src/pages/HelpSupport/HelpSupport.tsx

```diff
@@ -1,22 +1,42 @@
-import { FC } from 'react'
-import SideNavigationBar from '@/components/SideNavigationBar'
-import Container from '@/layouts/Container/Container'
-import PageContent from '@/components/PageContent/PageContent'
-import ContactPage from './components/ContactPage/ContactPage'
-import Table from './components/ContactPage/components/Table'
-import QAHeader from './components/ContactPage/components/Table/components/QAHeader'
+import { FC, useState } from 'react';
+import SideNavigationBar from '@/components/SideNavigationBar';
+import Container from '@/layouts/Container/Container';
+import PageContent from '@/components/PageContent/PageContent';
+import ContactPage from './components/ContactPage/ContactPage';
+import Table from './components/ContactPage/components/Table';
+import QAHeader from './components/ContactPage/components/Table/components/QAHeader';
 
-const HelpSupport: FC = () => (
-  <Container>
-    <div className="flex text-xl min-h-[832px]">
-      <SideNavigationBar />
-      <PageContent>
-        <QAHeader />
-        <Table />
-        <ContactPage />
-      </PageContent>
-    </div>
-  </Container>
-)
+const HelpSupport: FC = () => {
+  const [pageHeight, setPageHeight] = useState(1100);
```

Comment:
```suggestion
  const [pageHeight, setPageHeight] = useState<int>(1100);
```

Reply:
yes I changed int to number, and resolved it